ddtrace/contrib/pylibmc/client.py

```diff
@@ -142,6 +143,7 @@ def _span(self, cmd_name):
         )
 
         span.set_tag_str(COMPONENT, config.pylibmc.integration_name)
+        span.set_tag_str(db.SYSTEM, memcached.SERVICE)
```

Comment:
While I see the benefit of reusing a constant that is already available, we end up in a situation then where a constant has two different purposes. wdyt?

Reply:
Fair, I'll add a new constant.